### PR TITLE
Add support for openshift oauth proxy

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**

### CODE REVIEW

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. ``docker pull stakater/forecastle:SNAPSHOT-PR-111-1``

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. ``docker pull stakater/forecastle:SNAPSHOT-PR-111-2``

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. ``docker pull stakater/forecastle:SNAPSHOT-PR-111-3``

File: deployments/kubernetes/templates/chart/values.yaml.tmpl [hunk 1]
```diff
@@ -23,6 +23,8 @@ forecastle:
     customApps: {}
   proxy:
     enabled: false
+  openshiftOauthProxy:
+		enabled: false
```
Comment (kahootali) [inline deployments/kubernetes/templates/chart/values.yaml.tmpl:27]: fix indentation

Comment (ahmedwaleedmalik) [reply deployments/kubernetes/templates/chart/values.yaml.tmpl]: Fixed

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. ``docker pull stakater/forecastle:SNAPSHOT-PR-111-4``

**Pull request merged**